apps/judicial-system/api/src/app/modules/defendant/models/defendant.model.ts

```diff
@@ -36,4 +36,19 @@ export class Defendant implements TDefendant {
 
   @Field({ nullable: true })
   readonly citizenship?: string
+
+  @Field({ nullable: true })
+  readonly defenderName?: string
+
+  @Field({ nullable: true })
+  readonly defenderNationalId?: string
+
+  @Field({ nullable: true })
+  readonly defenderEmail?: string
+
+  @Field({ nullable: true })
+  readonly defenderPhoneNumber?: string
+
+  @Field({ nullable: true })
+  readonly defendantWaivesRightToCounsel?: boolean
```

Comment:
I think this flag should be required. We don't have 3 states here. It is good to have it optional in the input and default to false.